src/errors.js

```diff
@@ -0,0 +1,77 @@
+const ERRORS = {
+  NOLOGGEDUSER: 'No logged user',
+  WRONGPARAMETER: 'One or several parameters are wrong',
+  WRONGPASSPHRASE: 'The given passphrase is wrong',
+  NOURLPROVIDED: 'No url provided',
+  EMPTYUSERLIST: 'User list is empty',
+  NOUSERNAME: 'No username provided',
+  NOPASSPHRASE: 'No passphrase provided',
+  EXISTINGUSERNAME: 'The username already exists',
+  EXISTINGURL: 'The application url already exists',
+  FUNCTIONNOTDEFINED: 'The function is not defined',
+  USERNAMENOTFOUND: 'The username does not exist',
+  NOEXISTINGKEY: 'The required key does not exist',
+  NOVALUE: 'No value provided',
+  NOTOKEN: 'Token does not exist',
+  BADTOKEN: 'Token is not valid',
+  BADREQUEST: 'Bad request',
+  NOTAUTHORIZED: 'Not authorized',
+  NOCRYPTOKEY: 'Not a cryptokey',
+  ENCRYPTIONERROR: 'Encryption error',
+  DECRYPTIONERROR: 'Decryption error',
+  NOIMPORTRULES: 'No import rules provided'
+}
+
+/**
+    * @typedef MasqError
+    * @type {Object}
+    * @property {boolean} error
+    * @property {int} status - THe status of the error
+    * @property {string} message - The status of the error
+    * @property {string} name - The name of the error
+    * @property {string} stack - The stack
```

Comment:
When is stack used ? we should/could use Error object to get stack

Reply:
Stack is not used, I delete it for now.